src/engine/strat_engine/thinpool.rs

```diff
@@ -483,18 +483,20 @@ impl ThinPool {
     #[allow(dead_code)]
     pub fn snapshot_filesystem(&mut self,
                                dm: &DM,
-                               filesystem_uuid: FilesystemUuid)
+                               origin_uuid: Uuid,
```

Comment:
I think renaming the param for clarity is fine, but can we keep the type as `FilesystemUuid`, or am I missing something?

Reply:
will fix.